examples/mnist/build.envd

```diff
@@ -9,7 +9,7 @@ def build():
         "numpy",
     ])
     shell("zsh")
-    config.jupyter(token="", port=8888)
```

Comment:
default port?

Reply:
The port is not needed, we will use a random port in the host to expose it.